polaris-tokens/README.md

```diff
@@ -10,25 +10,42 @@ Design tokens originated at Salesforce, and the best way to describe them is to
 
 ## Installation
 
-Run the following command using [npm](https://www.npmjs.com/):
-
 ```bash
 npm install @shopify/polaris-tokens
 ```
 
-If you prefer [Yarn](https://yarnpkg.com/en/), use the following command instead:
+## Usage
 
-```bash
-yarn add @shopify/polaris-tokens
-```
+#### Javascript
 
-## Usage
+Accessing all of the available token groups
 
 ```js
 import {tokens} from '@shopify/polaris-tokens';
 
-// Access all of the available token groups
-console.log(tokens);
+console.log(tokens.motion);
+```
+
+#### CSS
+
+Importing all of the css variables. CSS variables are prefixed with `--p` to signal that these variables
+
+```js
+import '@shopify/polaris-tokens/dist/css/styles.css';
+
+div {
+  background: var(--p-background);
+}
+```
+
+#### JSON
+
+Accessing a specific token group file via the dist folder
+
+```js
+const spacing = fs.readFile(
+  'node_modules/@shopify/polaris-tokens/dist/json/spacing.json',
+);
```

Comment:
```suggestion
const spacing = require('@shopify/polaris-tokens/dist/json/spacing.json');
```

While this would work I suggest promoting how we recommend access the JSON tokens.

Reply:
We could improve this experience even more by updating the package.json `exports` key:

```jsonc
// package.json
{
  "exports": {
    ".": {...standardOverrides},
    "/json/*": "/dist/json/*",
    "/css/*": "/dist/css/*
  },
}
```

Which would should enable the following API:
```js
const spacing = require('@shopify/polaris-tokens/json/spacing.json');
```

Reply:
Here are the Node docs if you decide to move forward with this approach:
https://nodejs.org/api/packages.html#exports-sugar